### DISCUSSION TITLE
read download progress from a file

### DISCUSSION THREAD

**Discussion by karabaralex** [transmission/transmission#3369] [Q&A]
In my setup it is easier to observe transmission status from filesystem, e.g. if incomplete folder has new file I'm considering that it start downloading or once new file appears in `finished` folder I consider torrent downloaded.

My questions are:
1. Is it possible to get torrent progress by reading some file in filesystem? Does transmission write somewhere?
2. Is it possible to delete torrent from transmission by removing some file somewhere? I tried to removing .torrent file from incomplete folder but it did not work

**Comment by Petrprogs** [comment]
Maybe try to use RPC?

**Reply by vchimishuk** [reply]
Yeah, RPC and `transmission-remote` exist just for this kind of tasks.
